lib/session-manager.js

```diff
@@ -18,9 +17,9 @@ function SessionManager (options) {
     throw new Error('options.port is required');
   }
 
-  if (!options.browser) {
-    throw new Error('options.browser is required');
-  }
+  //if (!options.browser) {
```

Comment:
You still need to correct the condition ;-) When using Appium and a native app, I set browser to `''` (empty string). So maybe we can change it to `if (!options.browser && options.browser !== '')`.